.github/workflows/publish-module.yml

```diff
@@ -71,3 +71,11 @@ jobs:
 
       - name: Publish module
         run: bicep publish "modules/${{ steps.parse-tag.outputs.module_path }}/main.json" --target "br:${{ secrets.PUBLISH_REGISTRY_SERVER }}/public/bicep/${{ steps.parse-tag.outputs.module_path }}:${{ steps.parse-tag.outputs.version }}" --documentationUri "${{ steps.parse-tag.outputs.documentation_uri }}" --force
+
+  publish-module-index:
+    needs: main
+    uses: ./.github/workflows/publish-module-index.yml
+    secrets:
+      PUBLISH_CLIENT_ID: ${{ secrets.PUBLISH_CLIENT_ID }}
+      PUBLISH_TENANT_ID: ${{ secrets.PUBLISH_TENANT_ID }}
+      PUBLISH_SUBSCRIPTION_ID: ${{ secrets.PUBLISH_SUBSCRIPTION_ID }}
```

Comment:
Unfortunately, we cannot publish module index right after publishing the module because there's a delay in synchronizing the modules to MCR from the ACR to which the module is published.

Reply:
One workaround is to add a job to poll the MCR API to see if the published tag is available before publishing module index.

Reply:
I could just set it up to run nightly, I guess.

Reply:
That works too.